source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,247 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about different **geospatial data** formats and how to
+search geospatial data with the MongoDB Go Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+Coordinates on Earth
+--------------------
+
+To store and query your geospatial data in MongoDB, use **GeoJSON**. GeoJSON is
+a data format created by the Internet Engineering Task Force (IETF). 
+
+Here is the location of MongoDB headquarters in GeoJSON:
+
+.. code-block:: json
+
+   "MongoDB Headquarters" : {
+      "type": "point",
+      "coordinates": [-73.986805, 40.7620853]
+   }
+
+For definitive information on GeoJSON, see the
+`official IETF specification <https://datatracker.ietf.org/doc/html/rfc7946>`__.
+
+.. external resource
+
+GeoJSON Positions
+~~~~~~~~~~~~~~~~~
+
+A position represents a single place on Earth and exists in code as an array
+containing the following values: 
+
+- Longitude in the first position (required)
+- Latitude in the second position (required)
+- Elevation in the third position (optional)
+
+.. important:: Longitude then Latitude
+
+  GeoJSON orders coordinates as longitude first and latitude second. This may
+  be surprising as geographic coordinate system conventions generally list
+  latitude first and longitude second. Make sure to check what format any other
+  tools you are working with use. Popular tools such as OpenStreetMap and Google
+  Maps list coordinates as latitude first and longitude second.
+
+GeoJSON Types
+~~~~~~~~~~~~~
+
+Your GeoJSON object's type determines its geometric shape. Geometric shapes are
+made up of positions.
+
+Here are some common GeoJSON types and how you can specify them with positions:
+
+- ``Point``: a single position. This could represent the location of a
+  `sculpture <https://en.wikipedia.org/wiki/Chicago_Picasso>`__. 
```

Comment:
Issue: The other two links are to images, while this is to a Wikipedia article.

Suggestion: Link to a picture instead.